### ISSUE TITLE
Accessibity improvements needed on the Front Page

### ISSUE THREAD

**Issue by miguelahonen** [NatLibFi/Skosmos#1036]
[Make the changes as shown in the illustration]

1) The Front Page should offer a possibility to tab through interactive items in a logical order [numbered order in the illustration].

2) The semantic structure of the page should be improved by relevant and comprehensive heading levels [as shown in the illustration]

3) Skip link is a shortcut to important parts of the page (content) and makes the access to content quicker. The skip link helps to avoid unnecessary repetition of static page elements [content area marked in the illustration, the skip link should be located in the beginning of the body tag]
![Finto-Frontpage](https://user-images.githubusercontent.com/50104570/90410977-bcd40200-e0b3-11ea-95e0-126116d53dd8.jpg)

**Comment by miguelahonen** [comment]
Check the title of the page. It should describe the page content as sortly and concisely as possible

Skip Link consists of the following sections of the page:
- Welcome to the Finto service!
- Available vocabularies and ontologies
- Tweets (Note: after the "Available vocabularies and ontologies")

Tab order:
- Language selection
- Ontology list
- Search area
- Top menu
- Interfaces
- Twitter

Headers levels (level H2, other header levels according to common sense order):
1) Available vocabularies and ontologies
2) Search (hidden header with the text "Search from all the ontologies")
3) Tweets

**Comment by miguelahonen** [comment]
The skip link should become visible when it receives keyboard focus. Maybe hiding it off screen and then to be positioned on screen when it receives keyboard focus?

**Comment by miguelahonen** [comment]
All the information for a screen reader should be available in Finnish, Swedish and English

**Comment by osma** [comment]
This is relevant for Skosmos 2 only.

We will make a new accessibility review for Skosmos 3, see #1871

Closing this issue as it is no longer relevant for current development.
